vector/src/main/java/im/vector/riotx/features/userdirectory/UserDirectoryViewState.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.riotx.features.userdirectory
+
+import androidx.paging.PagedList
+import arrow.core.Option
+import com.airbnb.mvrx.Async
+import com.airbnb.mvrx.MvRxState
+import com.airbnb.mvrx.Uninitialized
+import im.vector.matrix.android.api.session.user.model.User
+
+data class UserDirectoryViewState(
+        val excludedUserIds: Set<String>? = null,
+        val knownUsers: Async<PagedList<User>> = Uninitialized,
+        val directoryUsers: Async<List<User>> = Uninitialized,
+        val selectedUsers: Set<User> = emptySet(),
+        val createAndInviteState: Async<String> = Uninitialized,
+        val directorySearchTerm: String = "",
+        val filterKnownUsersValue: Option<String> = Option.empty(),
+        val title: String,
+        val menuResId: Int?
```

Comment:
I thought default value was mandatory for MvRxStates... Anyway title and menuResId should only be present in the args of the Fragment, as it is pure static UI